modules/carto/src/layers/carto-tile-layer.ts

```diff
@@ -67,7 +67,10 @@ const CartoTileLoader: LoaderWithParser = {
   id: 'cartoTile',
   module: 'carto',
   extensions: ['pbf'],
-  mimeTypes: ['application/x-protobuf'],
+  mimeTypes: [
+    'application/vnd.carto-vector-tile',
```

Comment:
Didn't you suggest `application/vnd.carto-tile`? I'm fine with this by the way

Reply:
It feels more specific and aligned with `carto-spatial-tile`